packages/manager/modules/pci/src/projects/project/kubernetes/details/service/reset-kubeconfig/reset-kubeconfig.controller.js

```diff
@@ -0,0 +1,51 @@
+import get from 'lodash/get';
+
+import { STATUS } from '../../constants';
+
+export default class kubernetesResetCtrl {
+  /* @ngInject */
+  constructor($translate, OvhApiCloudProjectKube) {
+    this.$translate = $translate;
+    this.OvhApiCloudProjectKube = OvhApiCloudProjectKube;
+  }
+
+  $onInit() {
+    this.isReseting = false;
+    this.isClusterReady = this.clusterStatus === STATUS.READY;
+  }
+
+  /**
+   * reset
```

Comment:
Adding some block comments is really appreciate but is it possible to  write a
description more explicit? Or remove it if not needed at all?

Reply:
Agreed. The piece of code was copied from another place which had this comment. It has been removed now. Thanks